lib/hyrax/configuration.rb

```diff
@@ -319,6 +319,12 @@ def active_deposit_agreement_acceptance
       @active_deposit_agreement_acceptance
     end
 
+    attr_writer :show_deposit_agreement
+    def show_deposit_agreement
```

Comment:
OK, I've updated to `show_deposit_agreement?` and created #508 for changing the rest of the boolean getter methods to also end with question marks.